<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsignarRutinas</class>
 <widget class="QWidget" name="AsignarRutinas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignar Dietas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="styleSheet">
      <string notr="true">background-color:#000000</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalTop">
        <item>
         <widget class="QTableWidget" name="tablaSolicitudes">
          <property name="styleSheet">
           <string notr="true">QTableWidget {
    background-color: #6d6d6d;
    color: white;
    gridline-color: #888888; 
    border: 1px solid #444444;
}

QTableWidget::item {
    background-color: #7d7d7d;
    border: 1px solid #555555;
    padding: 4px;
}

QTableWidget {
    background-color: #6d6d6d;
    color: white;
    gridline-color: #888888; 
    border: 1px solid #444444;
}

QTableWidget::item {
    background-color: #7d7d7d;
    border: 1px solid #555555;
    padding: 4px;
}

QHeaderView::section {
    background-color: #5d5d5d;
    color: white;
    border: 1px solid #444444;
    padding: 4px;
}

QTableCornerButton::section {
    background-color: #6d6d6d;
    border: 1px solid #555555;
}


QScrollBar:vertical {
    background: #5d5d5d; 	
    width: 12px;
    margin: 0px;
    border: none;
}

QScrollBar::handle:vertical {
    background: #888888; 	
    min-height: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
    background: none;
    border: none;
}

QScrollBar:horizontal {
    background: #5d5d5d;
    height: 12px;
    margin: 0px;
    border: none;
}

QScrollBar::handle:horizontal {
    background: #888888;
    min-width: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    width: 0px;
    background: none;
    border: none;
}

QTableWidget QLineEdit {
    background-color: #5d5d5d;
    color: white;
    border: 1px solid #777777;
    padding: 2px;
}</string>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <column>
           <property name="text">
            <string>ID Solicitud</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID Socio</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Objetivo</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Fecha Solicitud</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tablaDietas">
          <property name="styleSheet">
           <string notr="true">QTableWidget {
    background-color: #6d6d6d;
    color: white;
    gridline-color: #888888; 
    border: 1px solid #444444;
}

QTableWidget::item {
    background-color: #7d7d7d;
    border: 1px solid #555555;
    padding: 4px;
}

QTableWidget {
    background-color: #6d6d6d;
    color: white;
    gridline-color: #888888; 
    border: 1px solid #444444;
}

QTableWidget::item {
    background-color: #7d7d7d;
    border: 1px solid #555555;
    padding: 4px;
}

QHeaderView::section {
    background-color: #5d5d5d;
    color: white;
    border: 1px solid #444444;
    padding: 4px;
}

QTableCornerButton::section {
    background-color: #6d6d6d;
    border: 1px solid #555555;
}


QScrollBar:vertical {
    background: #5d5d5d; 	
    width: 12px;
    margin: 0px;
    border: none;
}

QScrollBar::handle:vertical {
    background: #888888; 	
    min-height: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
    background: none;
    border: none;
}

QScrollBar:horizontal {
    background: #5d5d5d;
    height: 12px;
    margin: 0px;
    border: none;
}

QScrollBar::handle:horizontal {
    background: #888888;
    min-width: 20px;
    border-radius: 4px;
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    width: 0px;
    background: none;
    border: none;
}

QTableWidget QLineEdit {
    background-color: #5d5d5d;
    color: white;
    border: 1px solid #777777;
    padding: 2px;
}</string>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nombre</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Descripción</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="formLayout">
        <item>
         <widget class="QLabel" name="labelNombre">
          <property name="styleSheet">
           <string notr="true">QLabel {
    background-color: #2b2b2b; 	 
    color: #ff4c4c; 	 	 	 	 	 
    border: 2px solid #cc0000; 	 
    padding: 4px;
    font-size: 12px;
    font-weight: bold;
    border-radius: 6px;
}

QLabel {
    text-decoration: underline;
}</string>
          </property>
          <property name="text">
           <string>Nombre Dieta:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputNombre">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: #000000;
    border: 1.5px solid #f0f0f0;
    color: #f0f0f0;
    padding: 4px;
    border-radius: 4px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDescripcion">
          <property name="styleSheet">
           <string notr="true">QLabel {
    background-color: #2b2b2b; 	 
    color: #ff4c4c; 	 	 	 	 	 
    border: 2px solid #cc0000; 	 
    padding: 4px;
    font-size: 12px;
    font-weight: bold;
    border-radius: 6px;
}

QLabel {
    text-decoration: underline;
}</string>
          </property>
          <property name="text">
           <string>Descripción:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="inputDescripcion">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
    background-color: #000000;
    border: 1.5px solid #f0f0f0;
    color: #f0f0f0;
    padding: 2px;
    border-radius: 4px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="btnCrear">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 6px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
            </property>
            <property name="text">
             <string>Crear Dieta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAsignar">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 6px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
            </property>
            <property name="text">
             <string>Asignar a Usuario</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEliminar">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 6px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
            </property>
            <property name="text">
             <string>Eliminar Dieta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancelar">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #da0000;
    color: white;
    border-radius: 10px;
    padding: 6px;
    font-size: 12px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #aa0000;
}</string>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
